syntax = "proto3";

package tesla.proto.energy_device.v1;

option go_package = "github.com/teslamotors/energy_device/pkg/protocol/protobuf/energydevice/v1";
option java_package = "com.tesla.generated.energy_device.v1";
option java_outer_classname = "ControlEventSchedulingInfo";

import "google/protobuf/timestamp.proto";

message ControlEventSchedulingInfo {
  google.protobuf.Timestamp start_time = 1;
  uint32 duration_seconds              = 2;
  uint64 priority                      = 3;
}